src/components/QualityEvaluation/QualityEvaluation.tsx

```diff
@@ -0,0 +1,84 @@
+/**
+ * Copyright (c) 2024-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import { useField } from "formik";
+import { CSSProperties, useMemo } from "react";
+import { useTranslation } from "react-i18next";
+import styled from "@emotion/styled";
+import { colors, spacing } from "@ndla/core";
+import { IQualityEvaluation } from "@ndla/types-backend/draft-api";
+import { Text } from "@ndla/typography";
+import { QualityEvaluationFormValues, gradeItemStyles } from "./QualityEvaluationForm";
+import QualityEvaluationModal from "./QualityEvaluationModal";
+
+const qualityEvaluationOptions: { value: number; color: string }[] = [
```

Comment:
Er ikke snakk om kritisk forskjell i ytelse, men sida denne brukes til å matche numerisk verdi fra 1-5 for å finne en farge, er det bedre å bare bruke et array med farger, og hente på index+1?

Det samme objektet ligger også i `QualityEvaluationForm.tsx`, så uansett hvordan det struktureres, kan det sikkert flyttes slik at det kun ligger i en av filene også eksporteres til den andre.

Reply:
Ups, gikk litt fort i refaktureringen, burde gjenbruke samme ja! 

Tanken med å eksplisitt legge inn en value var for å tydeliggjøre at fargene er knyttet til en tall-verdi, men åpen for å endre til å bruke index-verdien også!